packages/server/lib/util/print-run.ts

```diff
@@ -553,3 +541,21 @@ export function displayVideoCompressionProgress (opts: { videoName: string, vide
     },
   }
 }
+
+export const printVideoHeader = () => {
+  console.log('')
+
+  terminal.header('Video', {
+    color: ['cyan'],
+  })
+}
+
+export const printVideoPath = (videoName?: string) => {
+  if (videoName !== undefined) {
+    console.log('')
```

Comment:
are these just page breaks? could you condense this to a '\n' in the main console log? (mostly just curious)

Reply:
pretty much just page breaks. We might be able to get these to `\n` but honestly the snapshots and console printing is so sensitive I am hesitant to touch it